src/main.rs

```diff
@@ -135,6 +139,20 @@ impl EntropySource for MessengerUtilities {
     }
 }
 
+impl Logger for MessengerUtilities {
+    fn log(&self, record: &Record) {
+        let args_str = record.args.to_string();
+        match record.level {
+            Level::Gossip => {}
```

Comment:
Wasn't totally sure what {} does in this line/context - does that just ignore the gossip messages? Is there any chance we might want to keep those logs?

Reply:
Indeed, it'll just drop. I figured that on `OnionMessenger` _probably_ won't get any gossip logs anyway so we can leave it, but I can update to trace if you think that's better?

Reply:
Yeah it doesn't really seem relevant IMO.